src/components/API.js

```diff
@@ -0,0 +1,41 @@
+import { useState, useEffect } from 'react';
+import './API.css';
+
+function Fetch() {
```

Comment:
[REQUIRED] One of the general principles of coding is descriptive naming. Kindly name this component including the filename `Quotes`